src/main/java/com/google/sps/servlets/AuthorsServlet.java

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Resources;
+import com.google.sps.data.Author;
+import com.hubspot.jinjava.Jinjava;
+import com.hubspot.jinjava.JinjavaConfig;
+import com.hubspot.jinjava.loader.FileLocator;
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(urlPatterns = {"/authors"})
+public class AuthorsServlet extends HttpServlet {
+
+  private static final Collection<Author> authors = new ArrayList<>(3);
+
+  @Override
+  public void init() {
+    authors.add(
+        new Author(
+            "Thomas Quintanilla",
+            "Duke University",
+            "I'm a kool boi. I do computer science things. I make cool mp3 and me project. "
+                + "I love mentorship, and I am a passionate boi.",
+            "thomas.jpg"));
+    authors.add(
+        new Author(
+            "Sylvia Ziyu Zhang",
+            "Carnegie Mellon University",
+            "I'm a kool gal. I do computer science things. I do cool natural language processing things. "
```

Comment:
Move all fake data to a FakeDataAccessor

Reply:
the author data is static and wont ever change since its just some info about the three of us. should i still add it to a mock data class?

Reply:
Why is it not in the template itself if it is static. Doesn't seem like you need to do this processing for every call.